README.md

```diff
@@ -27,6 +27,8 @@ To get started:
     git clone https://github.com/Microsoft/ComputerVision
     ```
 1. Install the conda environment, you'll find the `environment.yml` file in the root directory. To build the conda environment:
+    > If you are on Windows, remove the line `- pycocotools>-2.0` and uncomment
```

Comment:
Fyi, I added some installation description in the object detection README, which after this PR will need to get edited or removed.

Reply:
maybe we should consider env yml generation like the reco repo in the future

Reply:
Yes. Let's discuss during our next 1:1. I know they have a script, which I guess means we'd have to have two: one for windows (.bat) and one for Linux (bash). Or maybe we can use setup.py. Not sure what the best option is

Reply:
Yes. One extra step, executing python scripts/generate_conda_file.py seems OK and it's easy to update out tests to work with that.